examples/compiled/bar_yearmonth_custom_format.vg.json

```diff
@@ -17,9 +17,14 @@
           "as": "yearmonth_date",
```

Comment:
Looking at the chart, it's really hard to know what the values for axis ticks are. Please revise this example. 

<img width="543" alt="Screen Shot 2019-07-29 at 9 30 21 PM" src="https://user-images.githubusercontent.com/589034/62076505-1fa7f080-b248-11e9-8a02-c2cf33a234aa.png">

Reply:
This PR doesn't change anything about the label.  

> it's really hard to know what the values for axis ticks are

This example is only there for explaining that you can customize the format. (See `format.md`.)  So if you strongly think this should be change, please submit a separate PR for it.